template/2-history.md

```diff
@@ -1,10 +1,7 @@
 # History
 
-*Maximum 1 page*
+The proof of concept (PoC) built at the end of sprint 10 contains many of the most important features: user authentication, posts (addition and deletion), comments (addition and deletion), votes, feed, profile picture, challenges, ranking and map.
 
-*Describe the V1 POC as planned by Sprint10.*
-
-*What did you learn?*
-
-*What is missing to bridge from PoC to MVP?*
+We already have many of the features we want in the PoC for the MVP. The two features that lack are groups and media support, which will be described in the section "The MVP".
 
+Moreover, the whole backend needs to be changed, for security reasons. The logic should not be done on the phone of the user—which could be malicious—but using cloud functions.
```

Comment:
I think it would be helpful to explain why this change improves security specifically.
So the reader isn't left wondering.

Reply:
Great idea! 
I added an explanation in bb5e3a740e49870237323078c9b2b8101d1f5922, tell me if it is clear enough.

Reply:
Yeah it's pretty good now :)
I left another comment for a suggestion, to see if we can further improve the explanation